app/src/main/java/com/example/stromtracker/ui/raeume/raeumeBearbeiten_Loeschen/RaeumeBearbeitenLoeschenFragment.kt

```diff
@@ -16,8 +19,19 @@ import com.example.stromtracker.ui.raeume.RaeumeViewModel
 
 
 //deklariert Raeumefragment als Unterklasse von Fragment
-class RaeumeBearbeitenLoeschenFragment(private var currRaum: Raum) : Fragment() {
+class RaeumeBearbeitenLoeschenFragment(
+    private var currRaum: Raum,
+    private var raumliste : ArrayList<Raum>
+) : Fragment() {
     private lateinit var raeumeViewModel: RaeumeViewModel
+    private lateinit var savebutton: View
+    private lateinit var abortbutton: View
+    private lateinit var deletebutton: View
+    private lateinit var informationfield: TextView
+    private lateinit var raumnameneditfeld: EditText
+    private var raumnameleer: String =
+        "Raum kann nicht gespeichert werden, da Kein Name eingegeben wurde."
+    private var raumnamesonstiges: String = "Raum - Sonstiges - darf nicht erstellt werden!"
```

Comment:
Auslagerung in strings.xml ?

Reply:
Hat leider nicht funktioniert.  Habe versucht mit R.string drauf zuzugreifen, habe aber nur int werte bekommen und als ich dann probiert habe mit to String zu casten, bekam ich eine Zahl ausgegeben.